src/test/responses/billing.getBillingCards.multiple.success.json

```diff
@@ -1,101 +0,0 @@
-{
```

Comment:
Putting this here for lack of a better place: can we also delete `src/test/responses/billing.processPayment.storageOnly.success.json`?

Reply:
Oh right, I was so caught up on deleting the BillingCardVO or whatever it was called that I totally forgot there were tests for other deleted endpoints!